plugins/vsphere/pkg/command/version.go

```diff
@@ -0,0 +1,40 @@
+// Copyright © 2020 The Knative Authors
```

Comment:
<!--copyright-bot-->
Missing copyright header:
```suggestion
/*
Copyright 2020 VMware, Inc.
SPDX-License-Identifier: Apache-2.0
*/

// Copyright © 2020 The Knative Authors
```

Reply:
I copied that from the example (see https://github.com/knative/client-contrib/blob/master/plugins/hello/pkg/command/version.go). Shouldn't I leave the existing header as-is?